Makefile

```diff
@@ -26,7 +26,7 @@ IMAGE_TAG_ARM := $(IMAGE_TAG)-arm
 # Name of the cover profile
 COVER_PROFILE := coverage.txt
 # Disable go sum database lookup for private repos
-GOPRIVATE=github.com/dapperlabs/*
+GOPRIVATE=github.com/onflow/*
```

Comment:
should this instead be `github.com/onflow/*-internal` so that it only disables for private internal repos?

Reply:
I hadn't considered that. I would need to test to make sure the regex works on the repo name, but I can do some testing to ensure that works as expected. Now that you mention it, I don't think we will want to disable that for all repos in the `onflow` org.